cypress/integration/homePageHeader.spec.js

```diff
@@ -0,0 +1,40 @@
+import { HomePage } from "../pom/homePage";
```

Comment:
Rename name spec file into: "header.spec.js", it will be your general spec file for all your test for header.